src/core/document.js

```diff
@@ -138,6 +139,14 @@ var Page = (function PageClosure() {
       return shadow(this, 'mediaBox', obj);
     },
 
+    get userUnit() {
+      var obj = this.getPageProp('UserUnit');
+      if (typeof obj !== 'number') {
```

Comment:
Please use the `isNum` utility function instead; note that you may have to add `var isNum = sharedUtil.isNum;` at top of the file along with the other `sharedUtil` imports.
Furthermore, since the specification mentions that this should be a positive number, let's add a check for that as well, e.g.
```javascript
if (!(isNum(obj) && obj > 0)) {
```